cmd/kube-controller-manager/app/plugins.go

```diff
@@ -95,32 +97,18 @@ func ProbeRecyclableVolumePlugins(config componentconfig.VolumeConfiguration) []
 	}
 	allPlugins = append(allPlugins, nfs.ProbeVolumePlugins(nfsConfig)...)
 
-	allPlugins = append(allPlugins, aws_ebs.ProbeVolumePlugins()...)
-	allPlugins = append(allPlugins, gce_pd.ProbeVolumePlugins()...)
-	allPlugins = append(allPlugins, cinder.ProbeVolumePlugins()...)
-	allPlugins = append(allPlugins, vsphere_volume.ProbeVolumePlugins()...)
-
-	return allPlugins
-}
-
-// NewVolumeProvisioner returns a volume provisioner to use when running in a cloud or development environment.
-// The beta implementation of provisioning allows 1 implied provisioner per cloud, until we allow configuration of many.
-// We explicitly map clouds to volume plugins here which allows us to configure many later without backwards compatibility issues.
-// Not all cloudproviders have provisioning capability, which is the reason for the bool in the return to tell the caller to expect one or not.
-func NewVolumeProvisioner(cloud cloudprovider.Interface, config componentconfig.VolumeConfiguration) (volume.ProvisionableVolumePlugin, error) {
 	switch {
```

Comment:
Looking at this closer the switch adds the volume type for a set cloud provider.  I think we should leave this until we've tested scenario's like ebs running in non aws cloud provider.